doc_admin/AdminManual.adoc

```diff
@@ -92,9 +95,9 @@ include::./oh_software_compatibility_matrix.adoc[]
 [#jre]
 ==== 3.1.2 Java Runtime Environment (JRE)
 
-Depending on the operating system, there are different ways to install a Java Virtual Machine and the Java Runtime Environment (JRE)
-
-For instance, on a Microsoft Windows® system, the JRE is downloaded and setup file is executed, while on Linux based systems a single command is typed into a terminal window.
+Depending on the operating system, there are different ways to install a Java Virtual Machine and the Java Runtime Environment (JRE).
+The oh.sh / oh.ps1 set of scripts can download and install the latest JRE available by downloading it from internet; it is also possibile to install
+manually, following the specific instructions for your operating system.
```

Comment:
If the plan is to try and remove all the `you` references maybe this should be `following the specific instructions for the operating system being used` are something similar

Reply:
Yes.